.meshrc.yaml

```diff
@@ -39,13 +39,20 @@ sources:
             responseSchema: ./json-schemas/snapshotBackground.json
             responseTypeName: Background
           - type: Query
-            field: BulkDownload
+            field: BulkDownloadDetails
```

Comment:
was this conflicting with some other field?  I think this will actually be the entity name in nextGen (`BulkDownload`) so would we want to leave it as is?

Reply:
One issue I can see would be that for graphQL, the `BulkDownload` schema field would accept an array of IDs, whereas our Rails endpoint is setup to just retrieve one, so we'd have to handle that in the resolver or update the Rails endpoint

Reply:
This was not returning a singular Bulk Download. But a Bulk Download plus other details. I changed it to make room for a future singular Bulk Download.

Reply:
ah I see now.  So this might just need to be a special snowflake endpoint for now, since it doesn't really fit with typical graphQL / nextGen schema?

Reply:
For now! It gets used in BulkDownloadList so I will take a look again in the next mini-project!